apiary-metastore-events/kafka-metastore-events/kafka-metastore-receiver/src/main/java/com/expediagroup/apiary/extensions/events/metastore/kafka/messaging/KafkaMessageReader.java

```diff
@@ -131,4 +83,52 @@ private void readRecordsIfNeeded() {
     }
   }
 
+  public static final class Builder {
+
+    private String bootstrapServers;
+    private String topicName;
+    private String groupId;
+    private String clientId = "ApiaryMetastoreReceiver";
+    private MetaStoreEventSerDe metaStoreEventSerDe = new JsonMetaStoreEventSerDe();
+    private Properties consumerProperties = new Properties();
+
+    private Builder(String bootstrapServers, String topicName, String groupId) {
+      this.bootstrapServers = bootstrapServers;
+      this.topicName = topicName;
+      this.groupId = groupId;
+    }
+
+    public static Builder aKafkaMessageReader(String bootstrapServers, String topicName, String groupId) {
```

Comment:
To me, this looks better:

```
KafkaMessageReader reader = KafkaMessageReader.Builder(bootstrapServers, topicName, groupId)
                                                    .build()
```

what do others think ?

Reply:
Plus, its done like this for SQS Receiver.

Reply:
I don't mind really, they're very similar. I have an intellij plugin which creates builders and it created it like this so went with it. But if we would prefer the other way I can change it.

Reply:
@massdosage ^^ which one do you prefer ?

Reply:
Updated this now, it's not _exactly_ the same as the sqs reader, but i think thats fine as that will be removed at some point anyway